runtime/src/consensus/tendermint/mod.rs

```diff
@@ -18,9 +18,10 @@ use crate::{
 };
 
 /// Tendermint consensus backend name.
-pub const BACKEND_NAME: &str = "tendermint";
+pub const BACKEND_NAME: &str = "cometbft";
```

Comment:
This likely means the upgrade test would fail as the old runtime is not aware of this change. Let's leave this identifier as `tendermint`.